vulnerabilities/importers/mozilla.py

```diff
@@ -0,0 +1,180 @@
+import re
+from typing import List
+from typing import Set
+
+import yaml
```

Comment:
Please use saneyaml instead which has some secure defaults (and that we maintain)

Reply:
Would we need to update all the importers to reflect this ?

Reply:
ok, enter a ticket separately then and let's do that separately.

Reply:
Done. Ticket here https://github.com/nexB/vulnerablecode/issues/440